addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComUndecodedRFMessage.java

```diff
@@ -104,13 +104,9 @@ public RFXComUndecodedRFMessage(byte[] message) throws RFXComException {
 
     @Override
     public String toString() {
+        String str = super.toString();
 
-        String str = "";
-
-        str += super.toString();
-        str += "\n - Sub type = " + subType;
-        str += "\n - Id = " + getDeviceId();
-        str += "\n - Message = " + DatatypeConverter.printHexBinary(rawMessage);
```

Comment:
This one is included in the `super.toString` and the `ID` just contained `UNDECODED` while the `packetType` is also `UNDECODED`